docs/introduction.rst

```diff
@@ -19,42 +31,82 @@ can be open source or proprietary (although some distribution services, like
 `Flathub <https://flathub.org/>`_, can have restrictions in this respect).
 
 The only technical requirements made by Flatpak are that applications follow a
-small number of Freedesktop standards, in order to enable desktop integration
+small number of FreeDesktop standards, in order to enable desktop integration
 (see :doc:`conventions`).
 
+Issues of current model of packaging
+------------------------------------
+
+It is important to understand the problems of the current model
+of packaging applications to understand the existence of Flatpak:
+
+- **Duplicated work packaging apps**: Many Linux distributions come with their own package
+  manager, package format and repository. This requires a lot of maintainers to package the
+  same application in various distributions, and/or the application developer to learn the
+  language of each format and then package the application in those distributions, and/or
```

Comment:
probably "or" would make more sense

Reply:
Which line exactly? I can't tell because GitHub isn't clear.

![image](https://user-images.githubusercontent.com/50847364/113231503-5f1eaf80-9269-11eb-9d4b-16926e3c0a37.png)